tests/conftest.py

```diff
@@ -73,6 +72,7 @@
 
 
 _LOGGER = logging.getLogger(__name__)
+TestClientGenerator = Callable[..., Coroutine[Any, Any, TestClient]]
```

Comment:
Considering how often this is used in the tests, I think it makes sense to add the `Alias`.
But maybe this is not the best place for it?

Reply:
We can't really use `AiohttpClient`, because that has hard-coded arguments
`AiohttpClient = Callable[[Union[Application, BaseTestServer]], Awaitable[TestClient]]`

Reply:
This isn't the best place for this. I would suggest introducing a typing module for tests (like we have for core itself as well).

Reply:
Moved to `tests/typing.py`

Reply:
Resolved in preliminary PR